.github/ISSUE_TEMPLATE/bug_8.yml

```
@@ -45,16 +45,16 @@ body:
     label: Environment
     description: |
       examples:
-        - **npm**: 7.6.3
-        - **Node**: 13.14.0
-        - **OS**: Ubuntu 20.04
-        - **platform**: Macbook Pro
+        - **`npm -v`**: **npm**: 7.6.3
+        - **`node -v`**: **node**: 13.14.0
+        - **OS Name**: Ubuntu 20.04
+        - **System Model Name**: Macbook Pro
         - **`npm config ls`**: `; "user" config from ...`
     value: |
         - npm:
-        - Node:
-        - OS:
-        - platform:
+        - node:
```

```suggestion
        - Node.js:
```
May as well use the proper name.